config/jobs/kubernetes-csi/csi-proxy/csi-proxy-config.yaml

```diff
@@ -32,55 +32,3 @@ presubmits:
         resources:
           requests:
             cpu: 2000m
-  - name: pull-kubernetes-csi-csi-proxy-integration
```

Comment:
It looks like this job was added to the generated csi-proxy-config.yaml by mistake in a prior PR.

Can you move it to a csi-proxy-manual-job-config.yaml file?